Add deployment monitor to dbt cloud deploy page (#3199)
dbt-labs/docs.getdbt.com: website/docs/docs/deploy/dbt-cloud-job.md

<Lightbox src="/img/docs/dbt-cloud/deployment/deploy-scheduler.jpg" width="100%" title="An overview of a dbt Cloud job run"/>

### Deployment monitor

> Use the Deployment monitor to gain greater visibility into your deployment health and quickly detect any issues with the improved filtering and sorting capabilities. You can now easily curate the view based on environment, prioritizing the ones that matter most, such as Production. With pre-filtered lists for in progress and in queue activities, it's quicker and easier to track the relevant activity and stay on top of your deployment status.

Reviewer: suggs: 

- first sentence reads like it's referring to the actual page/view in the UI which is shown in the screenshot below. sugg matching capitalization and bolding it since it's a UI label
- conciseness 
- https://www.merriam-webster.com/dictionary/prefilter -- merriam webster is my personal source of truth for American English spellings

```suggestion
Use the **Deployment Monitor** view for greater visibility into your deployment health and to quickly detect any issues with the filtering and sorting capabilities. You can curate the view based on the environment, such as production. With prefiltered lists for in progress and in queue activities, it's quicker and easier to track the relevant activity and your deployment status.
```

> The Deployment monitor is accessible by going to the main dbt Cloud dashboard page or clicking the dbt logo on the upper-left. It includes:

Reviewer: ```suggestion
The **Deployment Monitor** is accessible from the main dbt Cloud dashboard page and by clicking the dbt logo on the upper-left. It includes:
```

> - A new run history visualization component to quickly scan a larger number of recent runs and get a better overview of overall health/activity 

Reviewer: suggs: 

- words like "new" or "improved" are great for release notes. for docs, sugg avoiding them so they don't have to be maintained later (remove "new" since it's no longer new after some months/years, etc.) and it also helps keep the content concise
- add period for consistency with the other list items

```suggestion
- A run history visualization component so you can quickly scan a larger number of recent runs and get an overview of overall health/activity. 
```

Author: good callout!

> - Prefiltered lists to display **In Progress**, **In Queue**, and **Latest Failed** jobs. 

Author: ```suggestion
- Prefiltered views added to display **In Progress**, **In Queue**, or **Latest Failed Runs**.
```

> - Pre-filtered lists have been added for In Progress, In Queue, and Latest Failed jobs. 

Reviewer: sugg: lower case capitalization since it's not a product name

however, if you were referring to the "cards/areas" in the Deploy Monitor view/page sugg matching case in the UI and bolding the words since they're UI labels

```suggestion
- Prefiltered lists for in progress, in queue, and latest failed jobs. 
```

Author: ok gotcha! that answers my question above!

> - Filter the page by environment, with the preference being stored for future use. 

Reviewer: ```suggestion
- Filter the page by environment, with the preference of storing it for future use. 
```

> - Lastly, the visualization at the top of the page now filters runs by trigger and displays the last 24 hours of runs, with a limit of 500.

Reviewer: ```suggestion
- The visualization at the top of the page filters runs by trigger and displays the last 24 hours of runs, with a limit of 500.
```

> <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/deploy-monitor.jpg" width="85%" title="The Deployment monitor dashboard displays overall deployment health and activity, allows you to filter the page by environment, and more" />

Reviewer: sugg: bold UI label

```suggestion
<Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/deploy-monitor.jpg" width="85%" title="The **Deployment Monitor** dashboard displays overall deployment health and activity, allows you to filter the page by environment, and more" />
```
